documentation/MeetingMinutes/2024/MOM-2024-02-14.md

```diff
@@ -0,0 +1,137 @@
+
+
+## Discussion on Consensus in Camara Working Groups
+
+Recapitulation of the previous discussion on this topic: Jesús and Diego said there was consensus achieved during a WG call that we continue working on https://github.com/camaraproject/IdentityAndConsentManagement/pull/113
+Herbert said that lazy-consensus is the Camara way. Shilpa said no consensus was achieved. 
+Jesús disagrees and clarifies from his view of what happened in the previous call that the WG participants were asked, considering the different opinions, if we could agree as a way forward to continue with the "self-contained" contribution (PR #113) with compromise to properly note any changes related to the reused standard. And there was support from Ericsson, GSMA, and no objection from others on the call. DT of course preferred the delta approach, but Axel mentioned that he wouldn't disagree with the WG as long as the changes to the standard are properly marked in the document.
+Axel proposed that consensus should be achieved on Github because not everybody can attend meetings. Diego said that is not the Camara way and all CAMARA subprojects actually make decisions during the calls, and active WG participants are well aware of this.
+Shilpa proposed that if there is an important decision suggested to be made at the next meeting, then this should be communicated a few days before the meeting. So, that people know about the importance of the next meeting and attend when otherwise they would not attend.
+We agreed that is must be clear whether consensus is achieved or not, and that we need to get better on this.
+
+## Discussion the Way Forward regarding the Profile
+
+Jesús recapitulated the last meeting where PR #113 was discussed and that consensus was achieved.
+Axel and Shilpa did not agree with consensus beeing achieved - see previous section.
+Axel said that he created the issue https://github.com/camaraproject/IdentityAndConsentManagement/issues/122 and kindly asked WG members to comment on the issue.
+Axel also created PR https://github.com/camaraproject/IdentityAndConsentManagement/pull/121 and asked people to review it.
+
+After some discussion we agreed that WG members comment on the issue and review the PR seeking resolution and a decision whether to use the DT proposal or the TEF proposal as our WG's policy document.
+Elizabeth suggested that if discussions on GitHub get too long, we would call for a side meeting, anounce it on the mailing this and seek consensus.
+
+Axel agreed with this proposal and no objections where heard.
+
+Axel asked Bjorn Hjelm for his/OIDF's opinion. Bjorn said that OIDF is here to help.
+
+We agreed that if neither the commenting on the issue nor the review of the PR lead to consensus, the WG seek a vote at TSC.
+
+
+## Discussion about offline-access
```

Comment:
```suggestion
## Discussion about offline-access and Refresh Token
```

Reply:
I added the link to the PR to the title

Reply:
If you mention the PR instead, then mention Refresh Token as is in the name of the PR, which actually matches my suggestion that this discussion about offline-access and Refresh Token

Reply:
@AxelNennker this one is pending, not sure if you saw it